### ISSUE TITLE
Transfer to bazel-contrib?

### ISSUE THREAD

**Issue by hobofan** [bazel-contrib/bcr-ui#4]
I'll try to bring up this project in one of the next rules author SIG meetings, and check with the SIG, to see if the rough direction of it is good, and whether (/at what point) it makes sense to transfer this repo to the `bazel-contrib` org.

**Update**
It has been discussed in the last SIG meeting, and the repository is to be transferred to the `bazel-contrib` organization.

## Todos pre-transfer

- [x] Add license
- [x] Add proper README

## Todos post-transfer

- [x] Adjust deployment to GH pages (pending on the decision which domain we will be hosting under)
- [ ] Hook up triggers in BCR

**Comment by cgrindel** [comment]
@hobofan Did you want me to add this to the Rules SIG project board?

**Comment by hobofan** [comment]
@cgrindel Just did that 👍

**Comment by cgrindel** [comment]
@hobofan I see that the outstanding task is to `Hook up triggers in BCR`. I wonder if this is a separate ticket given that [the bcr-ui repository](https://github.com/bazel-contrib/bcr-ui/) now exists under `bazel-contrib`. What do you think?

**Comment by hobofan** [comment]
@cgrindel Yup, I split it out as #20
